content/200-concepts/200-database-connectors/08-cockroachdb.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'CockroachDB'
+metaTitle: 'CockroachDB database connector (Reference)'
+metaDescription: 'This page explains how Prisma can connect to a CockroachDB database using the CockroachDB database connector.'
+preview: true
+---
+
+<TopBlock>
+
+The CockroachDB data source connector connects Prisma to a [CockroachDB](https://www.cockroachlabs.com/) database server.
+
+</TopBlock>
+
+## Example
+
+To connect to a CockroachDB database server, you need to configure a [`datasource`](/concepts/components/prisma-schema/data-sources) block in your [Prisma schema file](/concepts/components/prisma-schema):
+
+```prisma file=schema.prisma
+datasource db {
+  provider = "cockroachdb"
+  url      = env("DATABASE_URL")
+}
+```
+
+The fields passed to the `datasource` block are:
+
+- `provider`: Specifies the `cockroachdb` data source connector.
+- `url`: Specifies the [connection URL](#connection-details) for the CockroachDB database server. In this case, an [environment variable is used](/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) to provide the connection URL.
+
+Because CockroachDB support is currently a preview feature, you also need to include it in the list of preview features in the [`generator`]() block:
```

Comment:
```suggestion
Because CockroachDB support is currently a Preview feature, you also need to include it in the list of preview features in the [`generator`]() block:
```